sos/report/plugins/openshift_ovn.py

```diff
@@ -27,6 +27,12 @@ def setup(self):
             "/var/log/openvswitch/libreswan.log",
             "/var/log/openvswitch/ovs-monitor-ipsec.log"
         ])
+        # Collect ovn interconnect specific files if exists.
+        self.add_copy_spec([
+            "/var/lib/ovn-ic/etc/ovnnb_db.db",
+            "/var/lib/ovn-ic/etc/ovnsb_db.db",
+            "/var/log/ovn-ic/acl-audit-log.log"
```

Comment:
why did you add this file? looks like we didn't collect it before

Reply:
oh! yes, removed it.